### ISSUE TITLE
Mini.files closing explorer after 1 second when using custom mappings.

### ISSUE THREAD

**Issue by JesperLundberg** [echasnovski/mini.nvim#1284]
### Contributing guidelines

- [X] I have read [CONTRIBUTING.md](https://github.com/echasnovski/mini.nvim/blob/main/CONTRIBUTING.md)
- [X] I have read [CODE_OF_CONDUCT.md](https://github.com/echasnovski/mini.nvim/blob/main/CODE_OF_CONDUCT.md)
- [X] I have updated 'mini.nvim' to latest version

### Module(s)

mini.files

### Description

After updating to latest version of the plugin I have problems with the explorer closing itself after maybe 1 second after moving one step in the tree (in or out).
I have remapped the keys so it looks like this:
```
	{
		-- File manager
		"echasnovski/mini.files",
		version = false,
		config = function()
			require("mini.files").setup({
				mappings = {
					go_in_plus = "<Right>",
					go_out_plus = "<Left>",
					synchronize = "<C-s>",
				},
			})
		end,
		event = "VeryLazy",
	},
```
This worked perfectly before I updated.

I tried looking at checkHealth to see if there were any conflicts but found nothing to explain this.

If I try using the default movements (h and l) it works as expected.

I tried using just go_in and go_out and it makes no difference.

### Neovim version

0.10.2

### Steps to reproduce

1. `nvim -nu minimal.lua`
2. ...

### Expected behavior

That using my mappings does not close the explorer until the last node is a file or I exit the explorer.

### Actual behavior

The explorer closing itself after maybe 1 second after moving one step in the tree (in or out).

**Comment by echasnovski** [comment]
Thanks for the issue!

I can not reproduce. With provided config any "in" and "out" movement works as expected. Closing after 1 second probably means that some window got closed when it wasn't supposed to.

I'd ask you for more information:
- Does any of `:nmap <Left>` or `:nmap <Right>` show anything not related to 'mini.files' *when executed inside active explorer*?
- Do you know the commit (either from 'mini.nvim' or standalone 'mini.files') at which you had everything work?
- Can you reproduce this with  *only* 'mini.files' set up?

**Comment by JesperLundberg** [comment]
I posted this and was going to edit with the reproduction steps (you are too quick to answer!) and... I can't reproduce it when I do as minimal as possible (using vanilla NvChad with just mini.files set up).

So, it seems there is something that is highjacking or also uses left and right that causes it.

The commit where it worked is this one: 1b5ada9d3a0df36cfbf7f50a314646c0673b8c87

I'll check out the :nmap on left and right. Thanks for the tip.
I also think we can close this as it's not an issue with mini.files as I can't reproduce with a clean slate.

**Comment by JesperLundberg** [comment]
I ran the nmap command on an open explorer and got the following:
![image](https://github.com/user-attachments/assets/e46143f7-6bd4-4d73-93f8-8ef3c60ebf2a)
and the same for Right but "Go in entry plus" instead of course.

I ran the nmap on a file with no explorer open and I get "No mapping found".

I'm thinking that maybe I want to start over with a clean NvChad and I'll lift everything over as if I just started from that one instead of all the upgrading I've done over the years.

**Comment by echasnovski** [comment]
> I posted this and was going to edit with the reproduction steps (you are too quick to answer!) and...

Sorry :shrug: :smile:

> The commit where it worked is this one: 1b5ada9d3a0df36cfbf7f50a314646c0673b8c87

Well, there was a change in how windows are handled, but I can't see how this can break custom mappings. Definitely not without reproduction steps, sorry :( And also mappings output seems to be ok.

------

Closing for now. Feel free to post reproduction steps here if you manage to find them.

**Comment by JesperLundberg** [comment]
...and I managed to fix the issue I had!

It was not related at all to mini.files. It was also not related to NvChad.

The issue was that my chadrc (settingsfile for NvChad) required the mappings file. It used to have to do that. Then a lot has changed and I do not need to require the mappings anymore (I realized this when I ran NvChad out of the box).

So I guess the file was read automatically and then I also tried requiring it.

This issue can be closed, buried and forgotten I think.

I just want to give massive thanks to you @echasnovski for the mini plugins, I use a few of them and they are awsome. Also that you're so darn quick to answer. <3
